fcrepo-http-forwarding/src/main/java/org/fcrepo/camel/httpforwarding/FcrepoHttpForwardingConfig.java

```diff
@@ -33,14 +33,14 @@
  * @author Demian Katz
  */
 @Configuration
-@Conditional(FcrepoHttpIndexerConfig.HttpIndexerEnabled.class)
-public class FcrepoHttpIndexerConfig extends BasePropsConfig {
+@Conditional(FcrepoHttpForwardingConfig.HttpForwardingingEnabled.class)
+public class FcrepoHttpForwardingConfig extends BasePropsConfig {
 
-    static final String HTTP_INDEXER_ENABLED = "http.indexer.enabled";
+    static final String HTTP_FORWARDING_ENABLED = "http.enabled";
```

Comment:
Thoughts on this being `httpforwarding.enabled`? Just `http.enabled` makes me think of a server but given there aren't any other http modules I don't think it matters too much.

Reply:
I say let's leave it. Demian put a video out that shows the setting as it currently is.